lib/MusicBrainz/Server/Data/Relationship.pm

```diff
@@ -466,13 +474,20 @@ sub _load_subset {
         }
     }
 
+    my $rel_ids_by_target_type;
+    if (ref($target_types) eq 'HASH') {
+        $rel_ids_by_target_type = $target_types;
+        $target_types = [keys %{$target_types}];
```

Comment:
Should this override `$target_types` silently, or should we somehow warn if both are passed at the same time?

Reply:
Oh. Duh. Yeah, ignore.